main.cpp

```diff
@@ -91,6 +91,13 @@ struct Point
     Point& multiply( const DoubleType& dt );
     Point& multiply( const IntType& it );
 
+    Point& operator*=( float m )
+    {
+        x *= m;
+        y *= m;
+        return *this;
+    }
+
```

Comment:
Why is this here?
- [ ] since it's here, use your `multiply(float)` function internally